### ISSUE TITLE
Slack Integration

### ISSUE THREAD

**Issue by tehisaac** [elarb/adidas-alerter#1]
This would be perfect if it had Slack, any way you can pop this in? Thank you for your work!

**Comment by elarb** [comment]
Can try to add it, what did you have in mind? "/" commands? An interactive bot?

**Comment by tehisaac** [comment]
Yes! Essentially to push notifications to a slack channel and then have interaction. For instance https://github.com/dzt/shopify-monitor

Did this with shopify and it is awesome. It would really take your scraper to a whole new level!
